tests/conftest.py

```diff
@@ -329,9 +329,7 @@ def response_mocker() -> ResponseMocker:
 async def setup_integration(hass: HomeAssistant, check_report_issue: None) -> None:
     ## Assert the string to ensure the format did not change
     if AwesomeVersion(HA_VERSION) >= "2023.11.0":
-        # Issues may be created because hacs accesses hass.components, hass.helpers and
-        # calls async_show_progress without passing a progress task
-        assert len(_async_suggest_report_issue_mock_call_tracker) in [0, 1, 2, 3]
+        assert not len(_async_suggest_report_issue_mock_call_tracker)
```

Comment:
This is bound to happen again. Maybe there should be constants like this which the fixture can check?
```
HA_DEV_MAX_ALLOWED_ISSUES = 0
HA_MIN_VERSION_MAX_ALLOWED_ISSUES = 0
```

Reply:
Once the current fixes are in, i think this can just be handled when it comes.
Can always add this later if needed.
But preferably it will always be 0, and a PR to fix it should be created instead of bumping this number